plugins/CoreHome/javascripts/broadcast.js

```diff
@@ -436,10 +440,6 @@ var broadcast = {
      *                       handler.
      */
     propagateNewPopoverParameter: function (handlerName, value) {
-        // init broadcast if not already done (it is required to make popovers work in widgetize mode)
-        if (broadcast.isWidgetizedDashboard()) {
-            broadcast.init(true);
-        }
```

Comment:
adding `<div piwik-popover-handler></div>` in the widgetized dashboard template makes this no longer required